File: app/src/main/java/br/com/ricardo/testeandroid/model/Year.java
package br.com.ricardo.testeandroid.model;

public class Year {

    private float fund;
    private float cdi;

    public Year(float fund, float cdi) {
        this.fund = fund;
        this.cdi = cdi;
    }

    public float getFund() {
        return fund;
    }

    public void setFund(float fund) {
        this.fund = fund;
    }

    public float getCdi() {
        return cdi;
    }

    public void setCdi(float cdi) {
        this.cdi = cdi;
    }
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/Year.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Year: Serializable {

    @SerializedName("fund")
    var fund: Double? = null
    @SerializedName("CDI")
    var cdi: Double? = null

    override fun toString(): String {
        return "Year(fund=$fund, cdi=$cdi)"
    }
}